src/scdaq-calo.conf

```diff
@@ -0,0 +1,90 @@
+################################################################################
+##
+## Input settings
+##
+################################################################################
+
+# Input settings, allowed values are:
+#   "wzdma"     for DMA driver from Wojciech M. Zabolotny
+#   "dma"       for XILINX DMA driver
+#   "filedma"   for reading from file and simulating DMA
+#   "file"      for reading from file
+input:wzdma
+#input:filedma
+
+## Settings for DMA input
+
+# DMA device
+dma_dev:/dev/xdma0_c2h_0
+
+# Max received packet size in bytes (buffer to reserve)
+dma_packet_buffer_size:1048576
+
+# Number of packet buffers to allocate
+dma_number_of_packet_buffers:1000
+
+# Print report each N packets, use 0 to disable
+packets_per_report:200000
+#packets_per_report:1
+
+## Settings for file input
+
+#input_file:/dev/shm/testdata.bin
+input_file:testdata.bin
+#input_file:../dumps/dump-empty-run.bin
+
+
+################################################################################
+##
+## Stream processor settings
+##
+################################################################################
+
+# Define processing type (unpacking)
+processor_type:PASS_THROUGH
+#processor_type:GMT
+
+# Enable software zero-supression
+doZS:no
+
+
+################################################################################
+##
+## Elastics processor settings
+##
+################################################################################
+
+port:8000
+elastic_url:http://something.somewhere
+pt_cut:7
+quality_cut:12
+
+
+################################################################################
+##
+## File output settings
+##
+################################################################################
+
+output_filename_prefix:scout_pass_through
+#output_filename_prefix:scout_gmt
+
+output_filename_base:/fff/BU0/ramdisk/scdaq
+max_file_size:8589934592
+
+# Always write data to a file regardless of the run status, useful for debugging
+output_force_write:no
+
+
+################################################################################
+##
+## SCDAQ Generic Settings
+##
+################################################################################
+
+# Pipeline settings
+threads:8
+
+enable_stream_processor:yes
+enable_elastic_processor:no
```

Comment:
Should these be moved to the respective configuration sections?

Reply:
done